phc-rails/app/controllers/api/v1/event_registrations_controller.rb

```diff
@@ -42,48 +39,54 @@ def search
   end
 
   def get_applied
-    @event_registration = EventRegistration.find_by(Number__c: params[:Number__c])
-    if !@event_registration.nil?
-      @services = @event_registration.services
-      render json: { status: "true", services: ["Acupuncture", "Haircuts", "Massage"] }
+    event_registration = EventRegistration.find_by(Number__c: params[:Number__c])
+    if !event_registration.nil?
+      applied_services = []
+      event_registration.services.each do |s|
+        if s.applied?
+          applied_services << s.name
+        end
+      end
+      render json: { status: "true", services: applied_services }
     else
       api_message_response(404, "Event registration with that number does not exist.")
     end
   end
 
   def update_service
-    registration = EventRegistration.find_by(Number__c: params[:Number__c])
-    if registration.nil?
+    event_registration = EventRegistration.find_by(Number__c: params[:Number__c])
+    if event_registration.nil?
```

Comment:
I might just be stupid but the unless clause sometimes confuses me.  I think it's more clear what is happening using .nil?